### PR TITLE
completed javascript-loops exercise

### PR DESCRIPTION
**Pull request by bbrown949**
Screenshot 2023-03-22 at 9 30 09 PM

### CODE REVIEW

**Comment by bbrown949** [comment]
better screenshot
Screenshot 2023-03-22 at 9 31 20 PM

File: javascript-conditionals/main.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+function isEven(number) {
+  if (number % 2 === 0) { console.log(true); } else { console.log(false); }
+}
+
+function startsWithJ(string) {
+  if (string[0] === 'J') { console.log(true); } else { console.log(false); }
+}
+function isOldEnoughToDrink(person) {
+  if (person.age >= 21) console.log(true);
+  else console.log(false);
```
Comment (shawnkost) [inline javascript-conditionals/main.js:35]: You're missing brackets (`{}`) after your `if` & `else` statements

File: javascript-conditionals/main.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+}
+
+function isOldEnoughToDrinkAndDrive(person) {
+  if (person >= 0) { console.log(false); }
+}
+
+function categorizeAcidity(pH) {
+  if (pH === 7) {
+    console.log('neutral');
+  } else if (pH > 0) {
```
Comment (shawnkost) [inline javascript-conditionals/main.js]: This should also be checking if it's less than 7

File: javascript-conditionals/main.js [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    console.log('acid');
+  } else if (pH <= 14) {
```
Comment (shawnkost) [inline javascript-conditionals/main.js]: This should also be checking if it's greater than 7

File: javascript-loops/main.js [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+}
+console.log('repeatedWord', repeatWord('Javascript', 10));
+
+function logEachCharacter(string) {
+  for (let i = 0; i < string.length; i++) {
+    console.log(string[i]);
+  }
+}
+console.log('logEachCharacter', logEachCharacter('Bryan Brown'));
+
```
Comment (shawnkost) [inline javascript-loops/main.js:57]: You're missing a function! You should have a function `doubleAll`

**Review by shawnkost — CHANGES_REQUESTED** [review]
Some changes requested here Bryan!! You still haven't submitted `javascript-conditonals`, so you'll need to make sure to also submit the `javascript-conditionals` code on that branch as well, so the exercise will get marked as complete. You can just download the starter files again, and copy all the current work you have, and paste it there.

**Comment by bbrown949** [comment]
Added doubleAll function on javascript-loops
Screenshot 2023-03-26 at 12 33 40 PM

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**